prisma/migrations/20240803190002_fix_type_and_add_price/migration.sql

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Since this PR hasn't been merged to master, better to avoid adding migration file that fix previous migration files which hasn't been merged to master.